packages/editor/src/utils/custom-functions/index.ts

```diff
@@ -213,5 +213,10 @@ export function getCustomFunctionsRuntimeUrl(): string {
 export function getAllowCustomDataForDataTypeAny(): boolean {
   const userSettings = getUserSettings();
 
-  return userSettings['allowCustomDataForDataTypeAny'];
+  if (userSettings['allowCustomDataForDataTypeAny'] != undefined) {
```

Comment:
Use typescript !==

Reply:
Is the change making this setting can be undefined? We did not check undefined before?

Reply:
the setting here is a user setting which will have no value if user don't set it manually. As we'd like to keep the functionality that user is able to set it manually to true or false. so if it is not undefined, we will use the value here, otherwise we will set it to true as default.

Reply:
A simpler way to express a default value when undefined is

 `return userSettings['allowCustomDataForDatTypeAny'} ?? true;`

Reply:
thanks for the suggestion, changed to this way.